src/app/pages/OnDemandAudioPage/__snapshots__/index.test.jsx.snap

```diff
@@ -705,7 +724,7 @@ exports[`OnDemand Radio Page  should match snapshot for AMP 1`] = `
           <div
             class="emotion-25 emotion-26"
             data-e2e="on-demand-image"
-            dir="ltr"
+            dir="rtl"
```

Comment:
Odd that this is changing, is this just because `EpisodeImage` is now taking `dir` as a prop and before it defaulted to `"ltr"`?

Reply:
Nice catch. I'll update that right away